test/inductor/test_compiled_autograd.py

```diff
@@ -118,6 +118,49 @@ def fn():
 
         self.check_output_and_recompiles(fn)
 
+    def test_inputs_aliasing_bytecode(self):
```

Comment:
Adding a test to cover the bytecode directly. The behavior of this PR is covered by all tests once we add the inputs list clear on aot side in next PR.